automation/taskcluster/decision_task.py

```diff
@@ -0,0 +1,166 @@
+        """)
+        .with_artifacts(
+            "/build/repo/fxa-client/sdks/android/library/build/outputs/aar/fxaclient-release.aar",
+            "/build/repo/logins-api/android/library/build/outputs/aar/logins-release.aar",
+            "/build/repo/components/places/android/library/build/outputs/aar/places-release.aar",
+        )
+        .create()
+    )
+
+def android_arm32_release(build_task):
+    return (
+        linux_build_task("Android (all architectures): build and release")
+        .with_env(BUILD_TASK_ID=build_task)
+        .with_dependencies(build_task)
+        .with_script("""
+            curl --silent --show-error --fail --location --retry 5 --retry-delay 10 https://github.com/mozilla/sccache/releases/download/0.2.7/sccache-0.2.7-x86_64-unknown-linux-musl.tar.gz | tar -xz --strip-components=1 -C /usr/local/bin/ sccache-0.2.7-x86_64-unknown-linux-musl/sccache
+            ./automation/taskcluster/curl-artifact.sh ${BUILD_TASK_ID} target.tar.gz | tar -xz
+            ./gradlew --no-daemon clean :fxa-client-library:assembleRelease :logins-library:assembleRelease :places-library:assembleRelease
+            python automation/taskcluster/release/fetch-bintray-api-key.py
+            ./gradlew bintrayUpload --debug -PvcsTag="${GIT_SHA}"
+        """)
+        .with_artifacts(
+            "/build/repo/fxa-client/sdks/android/library/build/outputs/aar/fxaclient-release.aar",
+            "/build/repo/logins-api/android/library/build/outputs/aar/logins-release.aar",
+            "/build/repo/components/places/android/library/build/outputs/aar/places-release.aar",
+        )
+        .with_scopes("secrets:get:project/application-services/publish")
+        .with_features("taskclusterProxy")
+        .create()
+        # Eventually we can index these releases, if we choose to.
+        # .find_or_create("build.android_release." + CONFIG.git_sha)
+    )
+
+
+def dockerfile_path(name):
+    return os.path.join(os.path.dirname(__file__), "docker", name + ".dockerfile")
+
+
+def linux_task(name):
+    return DockerWorkerTask(name).with_worker_type("application-services-r")
+
+
+def linux_build_task(name):
+    return (
+        linux_task(name)
+        # https://docs.taskcluster.net/docs/reference/workers/docker-worker/docs/caches
+        .with_scopes("docker-worker:cache:application-services-*")
+        .with_caches(**{
+            "application-services-cargo-registry": "/root/.cargo/registry",
+            "application-services-cargo-git": "/root/.cargo/git",
+            "application-services-sccache": "/root/.cache/sccache",
+            "application-services-gradle": "/root/.gradle",
+            # After we get the docker-in-docker image building working, we can
+            # do this instead of baking Rust into our images.
+            # "application-services-rustup": "/root/.rustup",
```

Comment:
Can you file an issue for this?

Reply:
This is tracked by [Bug 1501720](https://bugzilla.mozilla.org/show_bug.cgi?id=1501720), but I've also filed https://github.com/mozilla/application-services/issues/339.